sdk/storage/Azure.Storage.DataMovement/src/StorageResource.cs

```diff
@@ -20,29 +21,34 @@ protected StorageResource()
         /// <summary>
         /// Defines whether the storage resource is a container.
         /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         protected internal abstract bool IsContainer { get; }
 
         /// <summary>
         /// Gets the Uri of the Storage Resource.
         /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public abstract Uri Uri { get; }
```

Comment:
I think Uri may be useful to leave so customers are able to identity their resource objects if they need to. I also sort of feel this way about `IsContainer` but we marked that as protected internal meaning we didn't want customers to use it so ehh